ipv6/ipv6.go

```diff
@@ -47,19 +47,34 @@ var _ = IPv6Describe("IPv6 Connectivity Tests", func() {
 		},
 	}
 
-	describeIPv6Tests := func(buildpackPath string, stack string) {
+	describeIPv6Tests := func(buildpackPath, stack, manifestFile string) {
 		appName = random_name.CATSRandomName("APP")
-		Expect(cf.Cf("push", appName,
-			"-m", DEFAULT_MEMORY_LIMIT,
-			"-p", buildpackPath,
-			"-s", stack,
-		).Wait(Config.DetectTimeoutDuration())).To(Exit(0))
 
-		for _, data := range EndpointTypeMap {
+		memoryLimit := DEFAULT_MEMORY_LIMIT
+		if manifestFile == "assets/java-spring/manifest.yml" {
+			memoryLimit = "1600MB" // Java Spring app specific memory limit
+		}
+
+		commandOptions := []string{"push", appName, "-m", memoryLimit, "-s", stack}
+		if manifestFile != "" {
```

Comment:
Wouldn't it be easier to add the path to the Spring app to the list of assets here:
https://github.com/cloudfoundry/cf-acceptance-tests/blob/cc7b2ce098a380008359b62a3d628681e1c19b54/helpers/assets/assets.go#L68
and then just use this folder for the `cf push -p` parameter?